### PR TITLE
BZ-1892592: adds specifying oc command to Ingress controller policy

### PR DESCRIPTION
**Pull request by JoeAldinger**
4.6+

[BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1892592)

[Preview](https://deploy-preview-44562--osdocs.netlify.app/openshift-enterprise/latest/networking/network_policy/about-network-policy.html)

Adds oc command to Ingress Controller policy to ensure outside traffic hits the endpoint pod.

@zhaozhanqi PTAL.

### CODE REVIEW

File: modules/nw-networkpolicy-about.adoc [hunk 1]
```diff
@@ -56,6 +56,16 @@ spec:
   policyTypes:
   - Ingress
 ----
++
+[TIP]
```
Comment (jeana-redhat) [inline modules/nw-networkpolicy-about.adoc:60]: I would consider this a warning based on the criteria here: https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

```suggestion
[IMPORTANT]
```

File: modules/nw-networkpolicy-about.adoc [hunk 2]
```diff
@@ -56,6 +56,16 @@ spec:
...
+====
+When using {product-title} Ingress Controller, you must apply the following policy to your project to ensure outside traffic reaches the endpoint pod:
```
Comment (jeana-redhat) [inline modules/nw-networkpolicy-about.adoc:62]:
```suggestion
When using the {product-title} Ingress Controller, you must apply the following policy to your project to ensure that outside traffic reaches the endpoint pod:
```

**Review by jeana-redhat — COMMENTED** [review]
Just a couple little things :)

**Comment by zhaozhanqi** [comment]
I do not think we still this changes,  the issue already be fixed in 4.7+ 
https://issues.redhat.com/browse/SDN-1340

cc @asood-rh

**Comment by bergerhoffer** [comment]
The enterprise-4.12 label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.11. And any PR going into `main` must also target the latest version branch (enterprise-4.12).

If the update in your PR does NOT apply to version 4.12 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Pull request closed without merging**